pkg/fuse/fuse.go

```diff
@@ -0,0 +1,113 @@
+package fuse
+
+import (
+	"os"
+	"strconv"
+
+	"context"
+	"github.com/golang/glog"
+	"github.com/kubevirt/device-plugin-manager/pkg/dpm"
+	pluginapi "k8s.io/kubernetes/pkg/kubelet/apis/deviceplugin/v1beta1"
+)
+
+const (
+	FUSEPath  = "/dev/fuse"
+	FUSEName  = "fuse"
+	Namespace = "devices.kubevirt.io"
+)
+
+type message struct{}
+
+type FusePlugin struct {
+	counter int
+	devs    []*pluginapi.Device
+	update  chan message
+}
+
+// object responsible for discovering initial pool of devices and their allocation.
+type FuseLister struct{}
+
+func (l FuseLister) GetResourceNamespace() string {
+	return Namespace
+}
+
+// Discovery discovers all FUSE devices within the system.
```

Comment:
This comment is missleading, it exposes single FUSE device from a predefined path. Please correct me if I'm wrong